shopfloor/services/cluster_picking.py

```diff
@@ -957,6 +957,7 @@ def set_destination_all(self, picking_batch_id, barcode, confirmation=False):
 
     def _unload_write_destination_on_lines(self, lines, location):
         lines.write({"shopfloor_unloaded": True, "location_dest_id": location.id})
+        lines.package_level_id.location_dest_id = location
```

Comment:
no fear of multiple recordset write issue?

Reply:
since 13.0, relations are automatically mapped